### PR TITLE
Begin development for v0.0.3

### PR DESCRIPTION
**Pull request by dfellis**

### CODE REVIEW

File: src/modules/index.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 export * as v0_0_1 from './0.0.1'
 export * as v0_0_2 from './0.0.2'
+export * as v0_0_3 from './0.0.3'
 export * as latest from './0.0.2' // Double-exported, but simplifies connect
```
Comment (aguillenv) [inline src/modules/index.ts:4]: When this will be updated?

Comment (aguillenv) [reply src/modules/index.ts:4]: Should we add the TODO if we will do it later?

**Review by aguillenv — APPROVED** [review]

File: src/modules/index.ts (hunk 1 above)
Comment (dfellis) [reply src/modules/index.ts:4]: Oh, actually let's do that now.

Comment (dfellis) [reply src/modules/index.ts:4]:
```suggestion
export * as latest from './0.0.3' // Double-exported, but simplifies connect
```

**Review by aguillenv — APPROVED** [review]

**Pull request merged**